test/uk/gov/hmrc/plasticpackagingtax/returns/views/returns/RecycledPlasticWeightViewSpec.scala

```diff
@@ -23,22 +23,31 @@ import uk.gov.hmrc.plasticpackagingtax.returns.base.unit.UnitViewSpec
 import uk.gov.hmrc.plasticpackagingtax.returns.controllers.returns.{routes => returnRoutes}
 import uk.gov.hmrc.plasticpackagingtax.returns.forms.RecycledPlasticWeight
 import uk.gov.hmrc.plasticpackagingtax.returns.forms.RecycledPlasticWeight.form
+import uk.gov.hmrc.plasticpackagingtax.returns.models.obligations.Obligation
 import uk.gov.hmrc.plasticpackagingtax.returns.views.html.returns.recycled_plastic_weight_page
 import uk.gov.hmrc.plasticpackagingtax.returns.views.tags.ViewTest
 
+import java.time.LocalDate
+
 @ViewTest
 class RecycledPlasticWeightViewSpec extends UnitViewSpec with Matchers {
 
   private val page = instanceOf[recycled_plastic_weight_page]
 
+  private val obligation = Obligation(LocalDate.of(2022, 4, 1),
```

Comment:
I think UnitViewSpec contains a defaultObligation. Up to you if you want to use it :)